src/swarm/neo/node/ConnectionHandler.d

```diff
@@ -693,12 +693,21 @@ class ConnectionHandler : IConnectionHandler
         assert(!(ci.flags & 8) && ci.defaultConstructor is null);
         assert(ci.destructor is null);
 
+        version (D_Version2)
+        {
+            auto initializer = ci.initializer();
+        }
+        else
+        {
+            auto initializer = ci.init;
```

Comment:
Huh, I had no idea this works in D1, was sure calling it with parens is required or `T.init` will be chosen. TIL.